docs/DeveloperGuide.adoc

```diff
@@ -353,6 +353,60 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
 
 Certain properties of the application can be controlled (e.g App name, logging level) through the configuration file (default: `config.json`).
 
+=== Bicycle Management
+
+As bicycles have a lot of properties, statuses and actions associated with them, they are represented in the code base as a dedicated `Bike` class.
+
+Each `AddressBook` will have its own list of `Bike`s, representing all the bicycles that a bicycle shop owner has at their disposal. For the most part, the list of bicycles will be maintained in the same way as the list of loans.
+
+Every bicycle will have the following members:
+* `name`: A name that uniquely identifies the bicycle, e.g. the bicycle's serial number.
+* `status`: The bicycle's current status that represents where it is or what it is doing, e.g. `Available`, `Loaned Out`, `On Display`, `Under Repair`.
+* `loanHistory`: A list of past loans associated with this bicycle, sorted chronologically.
+* `loanReserves`: A list of future loans (i.e. reservations) associated with this bicycle, sorted chronologically.
+* Traits that characterize the bicycle, such as `brand`, `model`, `top speed`, `wheel size`, `weight`, `material` etc.
```

Comment:
Line 363 to Line 367 are in the same paragraph instead of bulleted form. Can you change them? Because it looks a little messy now.

Reply:
Isn't this the correct format for a bulleted list?

Reply:
I sent the screenshot to you on telegram. I think you need a + at the end of each line.

Reply:
Fixed.